internal/datacoord/compaction.go

```diff
@@ -24,6 +24,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/errors"
+	"github.com/golang/protobuf/proto"
```

Comment:
please use `google.golang.org/protobuf/proto` instead
code check failed to detect this problem

Reply:
has fixed it.